### PR TITLE
2024.04 updates

### PR DESCRIPTION
**Pull request by MalarsriA**

### CODE REVIEW

**Comment by miladbb** [comment]
It's good to have a tag based on 2024.03-LTS before upgrading since it is the latest LTS

File: development/docker-compose/docker-compose.yaml [hunk 1]
```diff
@@ -153,10 +153,10 @@ services:
       JAVA_TOOL_OPTIONS: "-Dquarkus.log.category.'\"com.backbase\"'.level=DEBUG \
                           -Dquarkus.log.category.'\"org.keycloak.protocol.oidc\"'.level=TRACE"
     volumes:
-      - ./scripts/HealthCheck.java:/tmp/HealthCheck.java
+      - ./exe/HealthCheck.jar:/tmp/HealthCheck.jar
```
Comment (miladbb) [inline development/docker-compose/docker-compose.yaml:156]: this does not work for identity, the .java file works as it is still on java 17

Comment (mostafarastgar) [reply development/docker-compose/docker-compose.yaml:156]: even though the base [image](https://github.com/Backbase/local-backend-setup/pull/16/files#diff-b77f5795e92f51483c4e21d687301e7eb9c9976ea76d32367bf099b7bb0bc11fR39) is with java21?

Comment (MalarsriA) [reply development/docker-compose/docker-compose.yaml:156]: yes , i changed it to java 21

Comment (miladbb) [reply development/docker-compose/docker-compose.yaml:156]: the problem was in my local

**Review by miladbb — APPROVED** [review]

**Pull request merged**